R/utils.R

```diff
@@ -12,8 +12,34 @@
 #' subdirectory and file(s) within the package.
 #' @note This is an internal function and not intended to be called directly.
 read_csv_file <- function(...) {
-  read.csv(
-    system.file("extdata", ..., package = "fortedata", mustWork = TRUE),
-    stringsAsFactors = FALSE
+  weak_as_tibble(
+    read.csv(
+      system.file("extdata", ..., package = "fortedata", mustWork = TRUE),
+      stringsAsFactors = FALSE
+    )
```

Comment:
See my above comment about `colClasses`. If we wanted to go that route, could be worth adding an argument to this like `.readcsv_opts = list()` so that:

```r
argv <- list(file = system.file(...), stringsAsFactors = FALSE)
argv <- modifyList(argv, .readcsv_opts)
do.call(read.csv, argv)
```

Reply:
Thanks; linking this to #11